superset-frontend/src/dashboard/components/AddSliceCard.jsx

```diff
@@ -41,6 +41,67 @@ const defaultProps = {
   lastModified: null,
 };
 
+const Styled = styled.div`
+  .chart-card {
+    border: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+    border-radius: ${({ theme }) => theme.gridUnit}px;
```

Comment:
Just a nit, but for the sake of brevity since theme is so widely used here you could just - add all the styles within 
```
${({ theme})} => `...`
```

Reply:
Great tip! Done! Thanks!

Reply:
Adding that to an in-progress best practices doc :)